base_agent/dialogue_objects/interpreter.py

```diff
@@ -14,6 +14,31 @@
 from filter_helper import FilterInterpreter
 
 from base_agent.base_util import ErrorWithResponse, NextDialogueStep
+from base_agent.task import ControlBlock
+from base_agent.memory_nodes import TripleNode, TaskNode
+from base_agent.condition import NTimesCondition
+
+
+class TaskListWrapper:
+    """
+    takes a callable that outputs a list (of tasks) and turns it into a callable
```

Comment:
The docstring should be rephrased to make the functionality clearer. Just looking at the class, I don't quite understand what the purpose of that is is and why we need it. Can we make the documentation clearer ?